backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -597,10 +596,17 @@ public UserInfo getCurrentUserInfo() {
   @AuthorizationConfiguration.RequirePermissionManageUsers
   public List<ApiUser> getUsersInCurrentOrg() {
     Organization org = _orgService.getCurrentOrganization();
-    final Set<String> orgUserEmails = _oktaRepo.getAllUsersForOrganization(org);
-    return _apiUserRepo.findAllByLoginEmailInOrderByName(orgUserEmails);
+    List<ApiUser> usersInOrg;
+    if (_featureFlagsConfig.isOktaMigrationEnabled()) {
+      usersInOrg = _dbAuthService.getUsersInOrganization(org);
+    } else {
+      final Set<String> orgUserEmails = _oktaRepo.getAllUsersForOrganization(org);
+      usersInOrg = _apiUserRepo.findAllByLoginEmailInOrderByName(orgUserEmails);
+    }
+    return usersInOrg;
   }
 
+  // To be addressed in #8108
   @AuthorizationConfiguration.RequirePermissionManageUsers
   public List<ApiUserWithStatus> getUsersAndStatusInCurrentOrg() {
```

Comment:
⚠️ holding off on updating this until this is addressed #8103

Then, this will be updated in #8108